test/CMakeLists.txt

```diff
@@ -196,6 +196,7 @@ if( iodaconv_bufr_ENABLED )
     testoutput/amdar_wmo_multi2.nc
     testoutput/gnssro_2020-306-2358C2E6.nc
     testoutput/buoy_wmo_multi.nc
+    testoutput/buoy_wmo_multi2.nc
```

Comment:
Ryan will know how to solve this problem but it appears you started with an earlier feature-branch of ```sprint-ioda-converters``` because a merge of some earlier branch was done and you didn't have it yet in yours.  Anything with reference to buoy should be ignored basically.

Reply:
Ignored how?
@rhoneyager what do you think?